### ISSUE TITLE
Atrribute Error while importing tez

### ISSUE THREAD

**Issue by AnuraagBeniwal2610** [abhishekkrthakur/tez#21]
when i try to import tez using - import tez. I am encountering this error
AttributeError: module 'torch.optim.lr_scheduler' has no attribute 'SAVE_STATE_WARNING'

**Comment by abhishekkrthakur** [comment]
which version of torch are you using?

**Comment by AnuraagBeniwal2610** [comment]
i am using 1.8.0, which was automatically installed when i installed tez

**Comment by abhishekkrthakur** [comment]
yeah, i might need to fix that. 1.7.1 should work fine. if you could use it, it would be great.

for 1.8, ill make a new release soon.

**Comment by AnuraagBeniwal2610** [comment]
Ok sure, thanks for such a fast reply. Loved your book btw as well as your videos.

**Comment by MrR0b0t-23** [comment]
I encountered this error
AttributeError: module 'torch.optim.lr_scheduler' has no attribute 'SAVE_STATE_WARNING'

To solve this error I simply removed "warnings.filterwarnings("ignore", message=torch.optim.lr_scheduler.SAVE_STATE_WARNING)"  line from tez/tez/model/model.py

After changing this my program worked properly.
